tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -64,7 +64,8 @@ var (
 	// hybridOverlaySubnet is an annotation applied by the cluster network operator which is used by the hybrid overlay
 	hybridOverlaySubnet = "k8s.ovn.org/hybrid-overlay-hostsubnet"
 	// hybridOverlayMac is an annotation applied by the hybrid overlay
-	hybridOverlayMac = "k8s.ovn.org/hybrid-overlay-distributed-router-gateway-mac"
+	hybridOverlayMac          = "k8s.ovn.org/hybrid-overlay-distributed-router-gateway-mac"
+	remotePowerShellCmdPrefix = "powershell.exe -NonInteractive -ExecutionPolicy Bypass "
```

Comment:
Please add a comment stating what this does and why its necessary

Reply:
All these variables can be re-used from the framework once #103 merges, that's why I didn't comment